index.html

```diff
@@ -418,7 +430,7 @@ <h3>United States Privacy Law</h3>
       <h3>Other Jurisdictions and Privacy Rights</h3>
       <p>
         GPC could potentially be used to indicate rights in other jurisdictions as well. For example, the
-        GDPR potentially affords data subjects the right to limit the sharing of personal information under
```

Comment:
Please don't mix semantic changes into a PR that mostly just adds links. It makes it hard to notice and vet the semantic changes.

Reply:
That's fair of course, but this isn't much of a semantic choice. "Potentially" is repeated two sentences in a row and the existence of this right isn't hypothetical by any measure. Happy to revert this change if you prefer but it's clear that this is a simple phrasing error.